packages/uicore/src/components/Accordion/Accordion.tsx

```diff
@@ -29,21 +32,35 @@ function AccordionPanel(
   props: AccordionPanelProps & AccordionPanelInternalProps,
   ref: React.Ref<HTMLDivElement>
 ): React.ReactElement {
-  const { summary, details, togglePanel, isOpen, id, collapseProps, addDomId } = props
+  const {
+    summary,
+    details,
+    togglePanel,
+    isOpen,
+    id,
+    collapseProps,
+    addDomId,
+    disabled,
+    panelClassName,
+    summaryClassName,
+    detailsClassName,
+    chevronClassName
+  } = props
 
   return (
     <div
       ref={ref}
       data-testid={`${id}-panel`}
-      className={cx(css.panel, props.panelClassName)}
+      data-disabled={disabled}
+      className={cx(css.panel, panelClassName)}
       data-open={isOpen}
       id={addDomId ? `${id}-panel` : undefined}>
-      <div data-testid={`${id}-summary`} onClick={togglePanel} className={cx(css.summary, props.summaryClassName)}>
+      <div data-testid={`${id}-summary`} onClick={togglePanel} className={cx(css.summary, summaryClassName)}>
```

Comment:
Not an issue with your PR, but while you're in the area, it would be great if we can make this a bit more accessible. A click handler on a `div` won't be found by a screen reader. We could swap this out for a `button`, or we could add `aria-role="button" and maybe work in `aria-expanded` to give a hint as to whether this will expand or collapse the panel.

Ideally we could replace most of the JS in this component and just use a `<details>` tag with some custom styling, but that's probably a step too far for this ticket.